app/models/effective/datatables/employer_datatable.rb

```diff
@@ -38,6 +37,7 @@ class EmployerDatatable < Effective::MongoidDatatable
           [@latest_plan_year.try(:enrolled_summary), @latest_plan_year.try(:waived_summary)].compact.join("/")
           }, :filter => false, :sortable => false
         table_column :xml_submitted, :label => 'XML Submitted', :proc => Proc.new {|row| format_time_display(@employer_profile.xml_transmitted_timestamp)}, :filter => false, :sortable => false
+        table_column :attestation_status, :label => 'Attestation Status', :proc => Proc.new {|row| row.employer_profile.employer_attestation.aasm_state.try(:titleize) }, :filter => false, :sortable => false
```

Comment:
Remove try on aasm_state state. we never run into scenario where aasm_state is nil.

instead check employer_attestation present or not.